src/core/Utils.js

```diff
@@ -516,7 +516,12 @@ function isDOMElement (obj) {
  */
 function findDOMElement (element) {
   if (typeof element === 'string') {
-    return document.querySelector(element)
+    const elements = [].slice.call(document.querySelectorAll(element))
+    if (elements.length > 1) {
+      return elements
+    }
+    return elements[0]
+    // return document.querySelector(element)
```

Comment:
Changing this function means other cases where a target selector exists more than once on the page also return an array. Maybe better to have a separate function that always returns an array?

Reply:
Yep. Added a separate function, `findAllDOMElements`.